conformance/tests/httproute-invalid-cross-namespace-parent-ref.go

```diff
@@ -37,6 +39,16 @@ var HTTPRouteInvalidCrossNamespaceParentRef = suite.ConformanceTest{
 		gwNN := types.NamespacedName{Name: "same-namespace", Namespace: "gateway-conformance-infra"}
 		routeNN := types.NamespacedName{Name: "invalid-cross-namespace-parent-ref", Namespace: "gateway-conformance-web-backend"}
 
+		t.Run("HTTPRoute should have an Accepted: false condition with reason NotAllowedByListeners", func(t *testing.T) {
```

Comment:
Maybe add a comment above this that clarifies that:

1. We expect all implementations running conformance tests to have full namespace visibility and when they are running in that mode, this condition needs to be set.
2. It is valid for controllers to run in modes where they have access to a subset of namespace(s). In that case, they are not obligated to populate this condition on routes they can't see.

Reply:
Ah, I hadn't considered that use case. If you think it's better to not merge this in order to continue to accommodate that use case, that's OK with me for now; otherwise, I'll add the comment you suggest. I imagine at some point we might need to incorporate that into a conformance profile (#1709)?

Reply:
Thanks @skriss! I think it's good to add this comment, just as long as we have a clarifying comment. I think even the implementations that were deployed with limited access by default would likely have a way to watch all namespaces for the sake of conformance tests, just want to make sure it's clear that's not required for every use of Gateway API.

Reply:
Comment added!